src/pages/channel/config/chips.ts

```diff
@@ -0,0 +1,22 @@
+export const FILTER_LIST = [
+  { level: 100, name: '약', key: 'weak', color: 'danger' },
+  { level: 200, name: '중', key: 'moderate', color: 'warning' },
+  { level: 300, name: '강', key: 'strong', color: 'success' },
+] as const;
+
+export const CHANNEL_TYPE_LIST = [
+  { key: 'private', name: '개인채팅' },
+  { key: 'group', name: '그룹채팅' },
+] as const;
+
+export const FILTER_NAME_MAP = Object.fromEntries(
+  FILTER_LIST.map(({ key, name }) => [key, name]),
+);
+
+export const FILTER_COLOR_MAP = Object.fromEntries(
+  FILTER_LIST.map(({ key, color }) => [key, color]),
+);
+
+export const CHANNEL_TYPE_NAME_MAP = Object.fromEntries(
+  CHANNEL_TYPE_LIST.map(({ key, name }) => [key, name]),
+);
```

Comment:
```suggestion
export const CHANNEL_TYPE: Record<ChannelType, string> = {
  private: '개인채팅',
  group: '그룹채팅',
};
```
위에 `CHANNEL_TYPE_LIST`를 삭제하고 위와 같이 수정하면 더 깔끔하게 작성 가능합니다!